### PR TITLE
Fixes #8151: Technique Editor - whitespace as separator in "File ensure keys -> values present" method

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/8151

### CODE REVIEW

**Comment by peckpeck** [comment]
This PR is not mergeable

File: tree/30_generic_methods/package_install_version_cmp_update.cf [hunk 1]
```diff
@@ -22,13 +22,14 @@
 # ```
 # methods:
 #     "any" usebundle => package_install_version_cmp_update("postgresql", ">=", "9.1", "verify", "false");
-# ```
+ ```
```
Comment (peckpeck) [inline tree/30_generic_methods/package_install_version_cmp_update.cf]: Why the comment removal ?

**Comment by peckpeck** [comment]
@VinceMacBuche ping

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
PR rebased

**Pull request merged**